examples/flutter_gallery/lib/demo/material/bottom_app_bar_demo.dart

```diff
@@ -160,28 +163,21 @@ class _BottomAppBarDemoState extends State<BottomAppBarDemo> {
         padding: const EdgeInsets.only(bottom: 88.0),
         children: <Widget>[
           const _Heading('FAB Shape'),
-
           _RadioItem<Widget>(kCircularFab, _fabShape, _onFabShapeChanged),
           _RadioItem<Widget>(kDiamondFab, _fabShape, _onFabShapeChanged),
           _RadioItem<Widget>(kNoFab, _fabShape, _onFabShapeChanged),
-
           const Divider(),
           const _Heading('Notch'),
-
           _RadioItem<bool>(kShowNotchTrue, _showNotch, _onShowNotchChanged),
           _RadioItem<bool>(kShowNotchFalse, _showNotch, _onShowNotchChanged),
-
           const Divider(),
           const _Heading('FAB Position'),
-
           _RadioItem<FloatingActionButtonLocation>(kFabEndDocked, _fabLocation, _onFabLocationChanged),
           _RadioItem<FloatingActionButtonLocation>(kFabCenterDocked, _fabLocation, _onFabLocationChanged),
           _RadioItem<FloatingActionButtonLocation>(kFabEndFloat, _fabLocation, _onFabLocationChanged),
           _RadioItem<FloatingActionButtonLocation>(kFabCenterFloat, _fabLocation, _onFabLocationChanged),
-
           const Divider(),
           const _Heading('App bar color'),
-
           _ColorsItem(kBabColors, _babColor, _onBabColorChanged),
```

Comment:
not something i feel very strongly about, but these blank lines definitely were intentionally inserted for readability and so removing them seems dubious.